pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -80,8 +88,8 @@ func NewDeployer(cfg types.Config, labels map[string]string) *Deployer {
 
 var sanityCheck = versionCheck
 
-// sanityCheck guarantees the kpt and kustomize versions are compatible with skaffold.
-func versionCheck(dir string) error {
+// versionCheck guarantees the kpt and kustomize versions are compatible with skaffold.
```

Comment:
We're not guaranteeing any more.
```suggestion
// versionCheck returns an error if the kpt and kustomize versions are not compatible with skaffold.
```